lib/plug/conn.ex

```diff
@@ -19,6 +19,7 @@ defmodule Plug.Conn do
   * `path_info` - the path split into segments, example: `["hello", "world"]`
   * `script_name` - the initial portion of the URL's path that corresponds to the application
     routing, as segments, example: ["sub","app"]. It can be used to recover the `full_path/1`
+  * `request_uri` - the requested URI path, example: `/trailing/and//double//slashes/`
```

Comment:
@josevalim can we name it somehow else (e.g. `request_path`)?. In most cases by `request_uri` I expect to see `absoluteURI`, however in this place we store only `abs_path` (according RFC). Other options are:
- `absolute_path`
- `abs_path`
What are your thoughts?